gcloud-java-examples/README.md

```diff
@@ -69,6 +69,23 @@ To run examples from your command line:
   $mvn exec:java -Dexec.mainClass="com.google.gcloud.examples.DatastoreExample" -Dexec.args="my_name delete"
   ```
 
+  Here's an example run of `DnsExample`.
+  
+  Note that you have to enable the Google Cloud DNS API on the [Google Developers Console][developers-console] before running the following commands.
+  Note that the example creates and deletes dns records of type A only. Operations with other record types are not implemented in the example.
+  ```
+  $mvn exec:java -Dexec.mainClass="com.google.gcloud.examples.DnsExample" -Dexec.args="create some-sample-zone elaborateexample.com. description"
```

Comment:
I think it'd be nice to order these commands such that if the user were to type in the commands in the same order, they'd see some results.  For example, perhaps we should create a zone, add some records, then call get/list.